.gitmodules

```diff
@@ -32,10 +32,10 @@
 
 [submodule "parallelio"]
 path = libraries/parallelio
-url = https://github.com/NCAR/ParallelIO
+url = https://github.com/NCAR/ParallelIO.git
```

Comment:
is the `.git` necessary? I thought GitHub did not care. If I enter the URL with the `.git`, GitHub removes it.

Reply:
@gold2718 github does not care, but it's easier on git-fleximod. When it looks for fxDONOTUSEurl.